packages/kbn-plugin-helpers/lib/plugin_config.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (root) {
     '{lib,public,server,webpackShims,translations}/**/*',
   ];
 
-  return Object.assign({
+  return ({ ...{
```

Comment:
__nit:__ maybe just:
```js
return {
    root,
    kibanaRoot: resolve(root, '../../kibana'),
    serverTestPatterns: ['server/**/__tests__/**/*.js'],
    buildSourcePatterns: buildSourcePatterns,
    skipInstallDependencies: false,
    id: pkg.name,
    pkg,
    version: pkg.version,
  ...config,
};
```

Reply:
We should ignore this change, and I should remove it. My editor is set to fix linting/prettier errors, so this was just one of those.